### ISSUE TITLE
[FR] Parallel Coordinate plot

### ISSUE THREAD

**Issue by KronosTheLate** [JuliaPlots/Makie.jl#1954]
It would be awesome to have some version of a parallel coordinate plot in Makie.
![image](https://user-images.githubusercontent.com/61620837/169096961-b2ea631f-9f34-454d-bf7c-bafe1e5f5c19.png)
Taken from https://docs.wandb.ai/ref/app/features/panels/parallel-coordinates.

I have seen something like this in a talk for one of JuliaHub's products I believe.

**Comment by jkrumbiegel** [comment]
I've once written an example of this somewhere on Discourse, couldn't find it right now. I'm not sure what your example here actually shows, it looks a bit messy :) But I guess the point is to have plots that span multiple axes right? How do the coordinates between two axes work, I think I had only seen straight-line versions before.

**Comment by KronosTheLate** [comment]
It simply looks like some ploynomial fit, or some smoothing. Maybie loess?. I do however think that it both looks better and conveys some extra information compared to the straight lines.

The example shows three different hyperparameter values, and the lines are coloured by the accuracy of the model.

I love how tt is e.g. instantly clear the the worst performers are found in the bottom half of "channels_one", and in the top half of "learning_rate", simultaneously.
